hipify_torch/hipify_python.py

```diff
@@ -46,6 +47,8 @@
 to their actual types."""
 PYTORCH_TEMPLATE_MAP = {"Dtype": "scalar_t", "T": "scalar_t"}
 
+# Custom mapping json file (default), if the file is not available hipify call doesn't process it.
+custom_mapping_file = "custom.json"
```

Comment:
Where would the user need to put this "custom.json" file? In their repo or inside the hipify_torch directory? Also, may I request that we rename the default name of this file to "custom_hipify_mappings.json" to make it a bit less generic of a name?

Reply:
Users need to keep the custom_hipify_mappings.json file in the project_directory from where the hipify function is being called. If users wish to use a different file they can use ```custom_map``` argument and pass in the file path of the JSON file.